test/basic-querying.test.js

```diff
@@ -653,7 +663,7 @@ describe('basic-querying', function() {
       });
     });
 
-    describe('geo queries', function() {
+    bdd.describeIf(connectorCapabilities.geoPoint, 'geo queries', function() {
```

Comment:
The comment https://github.com/strongloop/loopback-datasource-juggler/pull/1810#discussion_r357700895 is applicable to this pull request too. Let's discuss the matter in the other PR first, and then apply the conclusions here.

> Hmm, I believe we are intentionally testing geo queries for all connectors, including those that don't provide native GeoPoint & near query support. IIRC, when the connector does not implement near queries, juggler will perform the search in memory.
> 
> Take `db2` and `dashdb` for an example. According to the check on the line 52 above, these connectors don't support GeoPoint type, yet they still pass all `describe('geo queries')` tests.
>
> Let's find a way how to allow the `ibmi` connector pass these test too.